NetworkPackageDemo/NetworkPackageDemo/Sources/NetworkPackage/example/XEndpoint.swift

```diff
@@ -11,6 +11,12 @@ enum XEndpoint {
 }
 
 extension XEndpoint: RequestBuilder {
+  
+
+    var bodyParamters: [String : Any] {
```

Comment:
it's duplication for the params, the user will get confused

Reply:
Done